packages/appeal-reply-service-api/.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "parserOptions": {
-    "ecmaVersion": 8,
+    "ecmaVersion": 2020,
```

Comment:
for optional chaining - discussed and approved, and in use elsewhere

only impacts linting